lib/tasks/devise_two_factor_migration.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :devise_two_factor do
+  desc 'Copy devise_two_factor OTP secret from old format to new format'
+  task migrate_encryption_format: [:environment] do
+    # Find user records 1_000 at a time
+    User.find_each do |user|
```

Comment:
We should only do it for users having an otp secret defined (2.5% of users on mastodon.social)

Reply:
Yup -- first pass I basically copied their migration script example directly. Will push update with conditions to limit to only relevant users.